log/log.go

```diff
@@ -18,6 +20,17 @@ func NewZapLogger() (*zap.SugaredLogger, error) {
 	loggerConfig.EncoderConfig.MessageKey = "message"
 	loggerConfig.EncoderConfig.EncodeTime = zapcore.EpochMillisTimeEncoder
 
+	// optionally override the log level from the default based on the LOG_LEVEL env var
+	lvl, exists := os.LookupEnv("LOG_LEVEL")
```

Comment:
@clairecng, I'm fairly fine with controlling this via a `LOG_LEVEL` env var, as that's a typical practice, but do we also want to pass this in via config, the way we are with injector log level?